data/settings/src/main/java/com/google/jetpackcamera/settings/model/TargetFrameRate.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2023-2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jetpackcamera.settings.model
+
+import android.util.Range
+
+enum class TargetFrameRate(val range: Range<Int>) {
```

Comment:
We shouldn't need this enum. We could just use `Int` directly. If you need to call out specific frame rates in code, you can put them in constants.